changelog/uda-function-arguments.dd

```diff
@@ -0,0 +1,10 @@
+UDAs on function arguments are now supported
+
+User-defined attributes on function arguments behave analogous to existing UDAs:
+
+---
+void test(A)(@(22) A a)
+{
+    static assert([__traits(getAttributes, a)] == [22]);
+}
+---
```

Comment:
Perhaps include an example of how to get the UDAs of a parameter from outside of a function. Or should that be left as an exercise for the reader 😃.

Reply:
That's a bit more complicated and I think we will provide an interface in `std.traits` anyhow.

Reply:
Ok.